cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -590,6 +590,22 @@ can then be used in a `SET` clause:
 MATCH (n)-[r:REL]->(m) SET n=properties(r)
 ----
 
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+MATCH (a), (b), allShortestPaths((a)-[r]->(b)) RETURN b
+
+MATCH (a), (b), shortestPath((a)-[r]->(b)) RETURN b
+----
+a|
+`shortestPath` and `allShortestPaths` without <<cypher-pattern-varlength, variable-length relationship>> have been deprecated. Instead, use a `MATCH` with a `LIMIT` of 1 or:
```

Comment:
```suggestion
`shortestPath` and `allShortestPaths` without <<cypher-pattern-varlength, variable-length relationship>> is deprecated. Instead, use a `MATCH` with a `LIMIT` of 1 or:
```

Reply:
I agree with the tense.
Would you prefer singular over plural though? Because there are two entities being deprecated, right?

Reply:
My bad, plural of course :)